pkgs/development/python-modules/pydot_ng/default.nix

```
@@ -16,13 +17,18 @@ buildPythonPackage rec {
     sha256 = "8c8073b97aa7030c28118961e2c6c92f046e4cb57aeba7df87146f7baa6530c5";
   };
 
-  buildInputs = [ pytest unittest2 ];
+  patches = [ ./0001-graphviz_path.patch ];
```

```suggestion
  patches = [ 
    (substituteAll {
      src = ./0001-graphviz_path.patch;
      graphviz = "${graphviz}/bin";
    })
  ];
```

This is an improved way to do the patching, it will eliminate the step in `postPatch`.